WordPress/Classes/ViewRelated/Views/BottomSheet/Demo/LocationResultTableViewCell.swift

```diff
@@ -0,0 +1,50 @@
+
+import UIKit
+
+// MARK: - LocationResultTableViewCell
+
+class LocationResultTableViewCell: UITableViewCell {
+
+    /// The designated cell reuse identifier
+    static var reuseIdentifier: String { return "\(self)" }
+
+    // MARK: UITableViewCell
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: .subtitle, reuseIdentifier: LocationResultTableViewCell.reuseIdentifier)
+        initialize()
+    }
+
+    required init?(coder aDecorder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func prepareForReuse() {
+        super.prepareForReuse()
+        initialize()
+    }
+
+    // MARK: Internal behavior
+
+    func setKey(key: String) {
```

Comment:
I imagine the call site of this method would read like `whatnot.setKey(key: "whatever")`.

Would it make sense to either rename it to `set(key: String)` or `setKey(_ key: String)` so it read a bit less redundant at the call site?